### PR TITLE
[2] Use webpack for dev server and build (new)

### PR DESCRIPTION
**Pull request by MatApple**
Reopening https://github.com/dcos/dcos-ui/pull/52 against feature branch.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

**Comment by MatApple** [comment]
@orlandohohmeier @mlunoe, had to close https://github.com/dcos/dcos-ui/issues/52 and reopen here against feature branch. I have made amendments based on your comments.

File: .build.config.js [hunk 1]
```diff
@@ -1,31 +0,0 @@
-var srcFolder = "./src";
```
Comment (mlunoe) [inline .build.config.js]: Could we `git mv` this file?

Comment (mlunoe) [inline .build.config.js]: Ok, just verified. There is no change to this file, so it shouldn't hold the PR back. But it would be nice to keep history

Comment (MatApple) [inline .build.config.js]: I did `git mv` on all moved files and don't understand why it hasn't been honored..

Comment (mlunoe) [inline .build.config.js]: Hmm... bad GitHub!

**Pull request merged**